### PR TITLE
optee-rust: fix building errors

### PR DESCRIPTION
**Pull request by DemesneGH**
Hi,
Teaclave TrustZone SDK 0.2.0 is released and the release note is in https://github.com/apache/incubator-teaclave-trustzone-sdk/releases/tag/v0.2.0.
Besides the release there are some fixes and updates, so `optee-rust` in OP-TEE repo is expected to be pinned to https://github.com/apache/incubator-teaclave-trustzone-sdk/commit/3272b38b013395e3376a38af6315633239d26c1c. I've opened a PR:https://github.com/OP-TEE/manifest/pull/211.
In the new version of Teaclave TrustZone SDK, OPTEE_DIR is checked in setup.sh. If OPTEE_DIR is not set, it will download OP-TEE source code. So set OPTEE_DIR as the directory of OP-TEE repo.

Best Regards,
Yuan

### CODE REVIEW

**Review by jbech-linaro — COMMENTED** [review]
`Acked-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by DemesneGH** [comment]
Please help to review the new commit, thanks!

Yuan

File: br-ext/package/optee_rust_examples_ext/optee_rust_examples_ext.mk [hunk 1]
```diff
@@ -21,6 +21,7 @@ export OPTEE_CLIENT_DIR = $(OPTEE_DIR)/out-br/build/optee_client_ext-1.0
 export OPTEE_CLIENT_INCLUDE = $(OPTEE_CLIENT_DIR)/out/export/usr/include
 export VENDOR = qemu_v8.mk
 export OPTEE_OS_INCLUDE = $(OPTEE_DIR)/optee_os/out/arm/export-ta_arm64/include
+export CC = $(OPTEE_DIR)/toolchains/aarch64/bin/aarch64-linux-gnu-gcc
```
Comment (jenswi-linaro) [inline br-ext/package/optee_rust_examples_ext/optee_rust_examples_ext.mk:24]: You only care about Aarch64?

Comment (DemesneGH) [reply br-ext/package/optee_rust_examples_ext/optee_rust_examples_ext.mk:24]: Yes. Rust TAs integrated into the OP-TEE buildroot are only aarch64. Aarch32 TAs are supported in Teaclave TrustZone SDK but not integrated into the OP-TEE buildroot yet.

Comment (jenswi-linaro) [reply br-ext/package/optee_rust_examples_ext/optee_rust_examples_ext.mk:24]: OK

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Comment by DemesneGH** [comment]
> Acked-by: Jens Wiklander <jens.wiklander@linaro.org>

Updated. Thanks!

**Comment by jforissier** [comment]
@DemesneGH FYI, in the OP-TEE projects at least, when someone gives a Acked-by: or Reviewed-by: without mentioning specific commits, it means you can apply the tag to all commits. So you can add Jens' tag to the first commit too. Thanks!

**Comment by DemesneGH** [comment]
@jforissier Updated, thanks!

**Pull request merged**